src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -91,114 +90,146 @@ public function negativeMatch($name, $subject, array $arguments)
     }
 
     /**
-     * @param callable $callable
-     * @param array    $arguments
-     * @param null     $exception
+     * @param callable           $callable
+     * @param array              $arguments
+     * @param null|object|string $exception
      *
      * @throws \PhpSpec\Exception\Example\FailureException
      * @throws \PhpSpec\Exception\Example\NotEqualException
      */
     public function verifyPositive($callable, array $arguments, $exception = null)
     {
-        try {
-            call_user_func_array($callable, $arguments);
-        } catch (\Exception $e) {
-            if (null === $exception) {
-                return;
+        $exceptionThrown = null;
+
+        if (PHP_VERSION_ID < 70000) {
```

Comment:
Yes, you can `catch` something that doesn't exist in the current PHP version without an error.